src/notation/internal/notationuiactions.cpp

```diff
@@ -349,13 +349,13 @@ const UiActionList NotationUiActions::m_actions = {
              QT_TRANSLATE_NOOP("action", "Edit score properties")
              ),
     UiAction("undo",
-             mu::context::UiCtxNotationOpenedOrTextEditing,
+             mu::context::UiCtxNotationFocused,
```

Comment:
@wizofaus 
Can you explain why you chose UiCtxNotationFocused over UiCtxNotationOpened here?
This is a global shortcut for score and doesn't need the notation to have focus.